packages/auto_mappr/lib/src/builder/map_model_body_method_builder.dart

```diff
@@ -70,6 +70,7 @@ class MapModelBodyMethodBuilder {
         ? mapping.whenSourceIsNullExpression!.returned
         : refer('Exception').newInstance([
             refer(
+              // ignore: avoid-adjacent-strings, to avoid parsing the errors in tests
```

Comment:
The comment does not describe why to ignore it here, no? Also, I would rather fix it here. (aka dont use adjacent strings)